vllm/engine/arg_utils.py

```diff
@@ -439,6 +440,11 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
                             type=int,
                             default=EngineArgs.tensor_parallel_size,
                             help='Number of tensor parallel replicas.')
+        parser.add_argument(
+            '--enable-expert-parallel',
+            action='store_true',
+            help='Use expert parallelism instead of tensor parallelism '
```

Comment:
more comment to say that EP is the multiplication of DP and TP?

Reply:
Actually I think we should leave this as-is and add this to the `--data-parallel` argument added in #https://github.com/vllm-project/vllm/pull/13923. We parallelize across all DP*TP ranks for the MoE layers regardless of if we are using EP.

Reply:
sounds good.